File: src/se/clau/intellijlux/psi/LuxElementType.java
package se.clau.intellijlux.psi;

import com.intellij.psi.tree.IElementType;
import se.clau.intellijlux.LuxLanguage;
import org.jetbrains.annotations.*;

public class LuxElementType extends IElementType {
  public LuxElementType(@NotNull @NonNls String debugName) {
    super(debugName, LuxLanguage.INSTANCE);
  }
}

File: src/se/clau/intellijlux/psi/LuxElementType.kt
package se.clau.intellijlux.psi

import com.intellij.psi.tree.IElementType
import org.jetbrains.annotations.NonNls
import se.clau.intellijlux.LuxLanguage

class LuxElementType(@NonNls debugName: String) :
  IElementType(debugName, LuxLanguage)
